hw/ip/rom_ctrl/dv/cov/rom_ctrl_cov_excl.el

```diff
@@ -112,3 +112,22 @@ Condition 34 "201396280" "(d_valid && d_error) 1 -1" (1 "01")
 // the request will not already have been popped from the request fifo, so reqfifo_rvalid must be
 // true.
 Condition 43 "721931741" "(rvalid_i & reqfifo_rvalid) 1 -1" (2 "10")
+
+// It is impossible to see !tl_i_int.a_valid & !error_internal. When a_valid is false, it is seen
+// in u_err inside tlul_adapter_sram.sv. This will make a_config_allowed inside u_err false as it
+// depends on addr_sz_chk, mask_chk and fulldata_chk. All of them are false if a_valid is false.
+// This means that in u_err, the first term of err_o becomes true.
+// Next, err_o become visible to error_det as tlul_error inside tlul_adapter_sram and it will
+// be true if tlul_error is true. error_det will then be seen as error_i to u_sram_byte inside
+// tlul_adapter_sram.
+// Since EnableIntg parameter is 0 for rom_ctrl, error_i comes out as error_o from u_sram_byte.
+// Then error_o would be seen as error_internal to tlul_adapter_sram and the conditional statement
+// is not able to cover 00 case for this reason.
```

Comment:
I'm not sure that 00 is right here. Should this be something like `0*1` ?

Reply:
For me, 00 was !a_valid and !error_internal. But replaced it with 01 to avoid confusion

Reply:
Oh, I'm really sorry. I was imagining three bits, where we don't have an opinion about the middle one. This would correspond to the three bits in the conditional coverage item. I don't think that `0*1` is a good way to write that though!

Maybe we should be really explicit, so end with something like "If error_o is true then it will cause error_internal to be true as well. As a result, if tl_i_int.a_valid is false then error_internal will be true, so we can't see the 011 condition (or 001 either)".